src/pages/journal/components/vehicle-details/__tests__/vehicle-details.spec.ts

```diff
@@ -27,35 +27,35 @@ describe('VehicleDetailsComponent', () => {
   });
 
   describe('DOM', () => {
-    it('should show vehicle details when showDimensions is true', () => {
-      component.showDimensions = true;
-      fixture.detectChanges();
-      const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
-      expect(ionCols.length).toBe(3);
-    });
-
-    it('should not vehicle details when showDimensions is false', () => {
-      component.showDimensions = false;
-      fixture.detectChanges();
-      const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
-      expect(ionCols.length).toBe(0);
-    });
-
-    it('should show number of seats when showNumberOfSeats is true', () => {
-      component.showDimensions = false;
-      component.showNumberOfSeats = true;
-      fixture.detectChanges();
-      const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
-      expect(ionCols.length).toBe(1);
-    });
-
-    it('should not show number of seats when showNumberOfSeats is false', () => {
-      component.showDimensions = false;
-      component.showNumberOfSeats = false;
-      fixture.detectChanges();
-      const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
-      expect(ionCols.length).toBe(0);
-    });
+    // it('should show vehicle details when showDimensions is true', () => {
+    //   component.showDimensions = true;
+    //   fixture.detectChanges();
+    //   const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
+    //   expect(ionCols.length).toBe(3);
+    // });
+
+    // it('should not vehicle details when showDimensions is false', () => {
+    //   component.showDimensions = false;
+    //   fixture.detectChanges();
+    //   const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
+    //   expect(ionCols.length).toBe(0);
+    // });
+
+    // it('should show number of seats when showNumberOfSeats is true', () => {
+    //   component.showDimensions = false;
+    //   component.showNumberOfSeats = true;
+    //   fixture.detectChanges();
+    //   const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
+    //   expect(ionCols.length).toBe(1);
+    // });
+
+    // it('should not show number of seats when showNumberOfSeats is false', () => {
+    //   component.showDimensions = false;
+    //   component.showNumberOfSeats = false;
+    //   fixture.detectChanges();
+    //   const ionCols = fixture.debugElement.queryAll(By.css('ion-col'));
+    //   expect(ionCols.length).toBe(0);
+    // });
   });
 
 });
```

Comment:
Commented out tests?  why?

Reply:
The vehicle details component no longer includes those elements and so yes, the tests should be removed actually